pihole-docker/step2.md

```diff
@@ -0,0 +1,53 @@
+# Setting up docker
+
+*Docker is available in this tutorial, but on your local device you must first install docker. Resources can be found [here](https://docs.docker.com/get-docker/).*
```

Comment:
potentially confusing. Assume we're on Katacoda, move this at the very end in an appendix